resources/js/app.js

```diff
@@ -4,18 +4,19 @@
 	app.config(['$routeProvider',
 		function($routeProvider) {
```

Comment:
also, your commit shortlog has no verb, making it hard to know what fix was done, could you rework it please? :)